salt/_modules/metalk8s_volumes.py

```diff
@@ -213,6 +213,25 @@ def clean_up(self):  # pragma: no cover
 
     def device_info(self):
         """Return size and path of the underlying block device"""
+        # NOTE: The ONLY case where `self.path` is not a "blkdev" is when we are using SparseLoop,
+        # which shouldn't be used in production so we do not consider those here (as we do not have
+        # any easy way to get the actual device path without the `/dev/disks/by-uuid` symlink)
```

Comment:
Does it seem acceptable to you ?

Reply:
We could find out the loop device associated with our sparse file using `losetup -j {self.path}`. Also, why do we scope the `udevadm trigger` to a single device? IIRC, if no device is specified, this command will just go through "its backlog of events".

Reply:
Agree that with losestup we can find it but a sparse file could be exposed as several devices so you may have several lines for it (even if I'm not sure it work well if we are in this case today)
We could trigger an udevadm on every device yes but I'm not sure about the impact on the device already mounted, right now I'm not fine running `udevadm trigger` on every device here as it will be triggered at any point in time when the platform is running

Reply:
As discussed offline let's do a simple global trigger on all loop when we have a problem with a sparse file